RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -737,6 +741,7 @@ HBHEPhase1Reconstructor::fillDescriptions(edm::ConfigurationDescriptions& descri
     desc.add<bool>("dropZSmarkedPassed");
     desc.add<bool>("tsFromDB");
     desc.add<bool>("recoParamsFromDB");
+    desc.add<bool>("saveEffectivePedestal");
```

Comment:
Sorry, but this is utterly useless as fillDescriptions as you do not provide a value for any of the parameters
(to be inserted if the parameter is not found in the py config).